apps/ichat-gw/Server.cxx

```diff
@@ -751,7 +751,7 @@ Server::checkSubscription(const std::string& to, const std::string& from)
    msg.addArg("sendSubscriptionResponse");
    msg.addArg(from.c_str());
    msg.addArg(to.c_str());   
-   msg.addArg(Data((unsigned long)success).c_str());   
```

Comment:
The change needed to be made as unsigned long caused ambiguity in the
function call. I added char * as it was shown to be a near match long back.
But, when I checked it now I think UInt32 is the most apt typecast. I'll
push change soon.

On Sun, Aug 21, 2016 at 1:02 PM, Daniel Pocock <notifications@github.com>
wrote:

> In apps/ichat-gw/Server.cxx
> <https://github.com/resiprocate/resiprocate/pull/57#discussion_r75594877>:
>
> > @@ -751,7 +751,7 @@ Server::checkSubscription(const std::string& to, const std::string& from)
> >     msg.addArg("sendSubscriptionResponse");
> >     msg.addArg(from.c_str());
> >     msg.addArg(to.c_str());
> > -   msg.addArg(Data((unsigned long)success).c_str());
>
> did you need to make this change? success is a bool, not a char*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/resiprocate/resiprocate/pull/57/files/d202b06b7dc3670a8a396f83f75e787cccdf231a#r75594877>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIukohCtKna_F2QZheTF8-QavL8eRle7ks5qiEyCgaJpZM4JpPdP>
> .
>



-- 


*Balram PariyarathTech Head | Excel 2016*
Computer Science and Engineering (Batch: 2013-17)
Govt. Model Engineering College, Kochi-21

[image: ☎]:  +91 8129130866
✉: balbbrl@gmail.com <p.balram96@gmail.com>